editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditFeaturesWithFeatureLinkedAnnotationSample.java

```diff
@@ -0,0 +1,288 @@
+import com.esri.arcgisruntime.geometry.GeometryEngine;
+import com.esri.arcgisruntime.geometry.GeometryType;
+import com.esri.arcgisruntime.geometry.Part;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.Polyline;
+import com.esri.arcgisruntime.geometry.PolylineBuilder;
+import com.esri.arcgisruntime.geometry.ProximityResult;
+import com.esri.arcgisruntime.layers.AnnotationLayer;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.IdentifyLayerResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class EditFeaturesWithFeatureLinkedAnnotationSample extends Application {
+
+  private MapView mapView;
+  private Geodatabase geodatabase; // keep loadable in scope to avoid garbage collection
+  private Feature selectedFeature = null;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add JavaFX scene to stage
+      stage.setTitle("Edit Features With Feature-linked Annotation Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create the map with a light gray canvas basemap centered on Loudoun, Virginia
+      ArcGISMap map = new ArcGISMap(Basemap.Type.LIGHT_GRAY_CANVAS_VECTOR, 39.0204, -77.4159, 18);
+
+      // create a map view and set its map
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // add the map view to stack pane
+      stackPane.getChildren().addAll(mapView);
+
+      // create and load the geodatabase
+      File geodatabaseFile = new File(System.getProperty("data.dir"),
+        "./samples-data/loudon/loudoun_anno.geodatabase");
+      geodatabase = new Geodatabase(geodatabaseFile.getAbsolutePath());
+
+      geodatabase.loadAsync();
+      geodatabase.addDoneLoadingListener(() -> {
+        if (geodatabase.getLoadStatus() == LoadStatus.LOADED) {
+          // create feature layers from tables in the geodatabase
+          var addressPointFeatureLayer = new FeatureLayer(
+            geodatabase.getGeodatabaseFeatureTable("Loudoun_Address_Points_1"));
+          var parcelLinesFeatureLayer = new FeatureLayer(
+            geodatabase.getGeodatabaseFeatureTable("ParcelLines_1"));
+          // create annotation layers from tables in the geodatabase
+          var addressPointsAnnotationLayer = new AnnotationLayer(
+            geodatabase.getGeodatabaseAnnotationTable("Loudoun_Address_PointsAnno_1"));
+          var parcelLinesAnnotationLayer = new AnnotationLayer(
+            geodatabase.getGeodatabaseAnnotationTable("ParcelLinesAnno_1"));
+
+          // add the feature layers and annotation layers to the map
```

Comment:
just some wordsmithing here:

```suggestion
          // add the feature and annotation layers to the map
```